components/Dialogs/examples/DialogsCustomShadowExampleViewController.swift

```diff
@@ -42,12 +42,23 @@ class CustomShadowViewController: UIViewController {
     bodyLabel.sizeToFit()
     self.view.addSubview(bodyLabel)
 
+    self.view.isAccessibilityElement = true
+    self.view.accessibilityTraits = .button
+    self.view.accessibilityLabel = bodyLabel.text
```

Comment:
Isn't that a really long string for the accessibility label? https://developer.apple.com/documentation/objectivec/nsobject/1615181-accessibilitylabel?language=objc

Reply:
I'm unsure. This is, technically the descriptive label as per "If an accessibility element does not display a descriptive label", and I'd assume that the dialog's text is meant to be read out here.

@romoore to weigh in here.